.opex/env/uat/config.yaml

```
@@ -0,0 +1,11 @@
+oa3_spec: ./api-spec/api.yaml # If start with http the file would be downloaded from the internet
+name: opex_pagopa-ecommerce-payment-requests-service
+location: West Europe
+timespan: 5m # Default, a number or a timespan https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/scalar-data-types/timespan
+data_source: /subscriptions/b9fc9419-6097-45fe-9f74-ba0641c91912/resourceGroups/pagopa-p-vnet-rg/providers/Microsoft.Network/applicationGateways/pagopa-u-app-gw
```

```suggestion
data_source: /subscriptions/b9fc9419-6097-45fe-9f74-ba0641c91912/resourceGroups/pagopa-u-vnet-rg/providers/Microsoft.Network/applicationGateways/pagopa-u-app-gw
```
This should point to UAT subscription id, right?